addons/hr_holidays/static/src/avatar_card_popover_patch.js

```diff
@@ -0,0 +1,17 @@
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
+
+import { getOutOfOfficeDateEndText } from "@hr_holidays/persona_model_patch";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(AvatarCardPopover.prototype, {
+    get fieldNames() {
+        return [...super.fieldNames, "out_of_office_date_end"];
+    },
+    get OutOfOfficeDateEndText() {
+        if (!this.user.out_of_office_date_end) {
+            return "";
+        }
+        return getOutOfOfficeDateEndText(this.user.out_of_office_date_end);
+    }
```

Comment:
getter should have named starting with capital letter (except if this returns class which is very rare and isn't the case here)